### ISSUE TITLE
A Toast Notification in quite bottom for showing the starting of Download and Bookmarked.

### ISSUE THREAD

**Issue by chaytanyasinha** [commons-app/apps-android-commons#3219]
Summary:
A small message just like the "ERROR" message should be shown when we start downloading and when we bookmarked any picture.
Steps to reproduce:
Just go to explore select a picture of your choice then just click on the download icon in the top right then it will be stored to your memory storage but there is no notification about the picture started to downloading
Just go to explore select a picture of your choice then just click on the star icon in the top right then it will be added to your bookmarks but there is no notification about the picture being added to bookmarks.

What did you expect the app to do, and what did you see instead?
App should show an notification about picture being added to bookmarks section or start of downloading and the toast notification should be shown in bottom.

Device-Samsung S6
Android-8.0
Would you like to work on the issue?
Yes
![WhatsApp Image 2019-11-03 at 23 50 19](https://user-images.githubusercontent.com/50200796/69496067-edf01f00-0ef3-11ea-91be-acc13e986de2.jpeg)

**Comment by ashishkumar468** [comment]
AFAIR, the download is handled by the download manager, does that not show a download progress notification.

**Comment by chaytanyasinha** [comment]
Yes @ashishkumar468 , it does show the download progress notification but I was talking about an toat notification like this in the screenshot.
![Uploading WhatsApp Image 2019-11-24 at 22.03.07 (1).jpeg…]()

**Comment by chaytanyasinha** [comment]
Yes @ashishkumar468 , it does show the download progress notification but I was talking about an toat notification like this in the screenshot.
![WhatsApp Image 2019-11-24 at 22 03 07 (1)](https://user-images.githubusercontent.com/50200796/69497871-9a87cc00-0f07-11ea-87e2-03c3d5b636c9.jpeg)

**Comment by nicolas-raoul** [comment]
I am not sure it is really needed, as the download notification appears, and most pictures are small enough that download will not take a long time.

**Comment by chaytanyasinha** [comment]
@nicolas-raoul I agree with you but many popular applications does use these type of toasts at starting of downloading so we can also try writing  "Downloading.."

**Comment by chaytanyasinha** [comment]
@ashishkumar468 @nicolas-raoul what do you think about it?

**Comment by neslihanturan** [comment]
I also think this is not needed at this stage. We have several of more important issues :)

**Comment by chaytanyasinha** [comment]
it's completely fine @neslihanturan .I'll be more happy to solve important issues.
